crypto3/libs/blueprint/test/zkevm_bbf/opcodes/pushx.cpp

```diff
@@ -93,6 +93,20 @@ BOOST_AUTO_TEST_CASE(pushx_strings) {
     max_sizes.max_zkevm_rows = 100;
     max_sizes.max_exponentiations = 50;
     max_sizes.max_exp_rows = 100;
+    max_sizes.max_filter_indices = 500;
+
+    if( circuits_to_run.empty() ) {
+        circuits_to_run.insert("zkevm");
+        circuits_to_run.insert("zkevm-wide");
+        circuits_to_run.insert("rw");
+        circuits_to_run.insert("bytecode");
+        circuits_to_run.insert("copy");
+        circuits_to_run.insert("logs");
```

Comment:
I think, it would be enough to switch on this test just for logx opcode tests and for one test without logs for CI performance.

It shows nothing interesting for other opcode tests.